xml/schema/logixng/logixng-4.23.1.xsd

```diff
@@ -230,6 +230,7 @@
               <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
               <xs:element name="comment" type="commentType" minOccurs="0" maxOccurs="1"/>
               <xs:element name="fileName" type="xs:string" minOccurs="1" maxOccurs="1"/>
+              <xs:element name="csvType" type="xs:string" minOccurs="0" maxOccurs="1"/>
```

Comment:
Please use enum instead of string here. Since you are using an enum in the Java code, there is no point in storing a string here. If someone wants to add new items to the enum, they can update the enum here as well. I have done that multiple times.